sdk/harambe/core.py

```diff
@@ -491,7 +491,10 @@ async def run(
 
             if not harness_options.get("disable_go_to_url", False):
                 response = await page.goto(url)
-                if response.status >= 400:
+                if (
+                    not harness_options.get("ignore_goto_error", False)
+                    and response.status >= 400
+                ):
```

Comment:
instead of ignore_goto_error could we pass an error handler callback?  I think this would give us a cleaner interface

ie:

if response.status >= 400:
   await error_handler_cb(response)
   
   
 the default cb would just reraise the erore